spec/requests/api/feeds_controller_spec.rb

```diff
@@ -30,4 +30,18 @@
       end
     end
   end
+
+  describe 'GET /api/feeds/favorites' do
+    let(:user) { create(:user) }
+
+    subject { get favorites_api_feeds_path, headers: user.create_new_auth_token }
```

Comment:
```suggestion
    subject { get favorites_api_feeds_path, headers: user.create_new_auth_token }

    let(:user) { create(:user) }
```

subjectは一番上に書いて欲しいですー